forest/shared/src/cli/config.rs

```diff
@@ -149,6 +149,8 @@ pub struct Config {
     pub client: Client,
     #[cfg(feature = "rocksdb")]
     pub rocks_db: forest_db::rocks_config::RocksDbConfig,
+    #[cfg(feature = "paritydb")]
+    pub parity_db: forest_db::parity_db::ParityDbConfig,
```

Comment:
Let's make it part of the configuration even if `paritydb` is disabled. That way, the configuration won't be invalid if you switch between databases.

Reply:
How exactly?

Reply:
By removing the `#[cfg(feature = "paritydb")]` line above it.